draft-ietf-acme-acme.md

```diff
@@ -1017,11 +1019,13 @@ encoded in the format specified in RFC 3339 {{!RFC3339}}.  This field is REQUIRE
 for objects with "valid" in the "status" field.
 
 challenges (required, array of objects):
-: For pending authorizations, the challenges that the client can fulfill in
-order to prove possession of the identifier.  For final authorizations (in the "valid" or "invalid" state), the
-challenges that were used.  Each array entry is an object with parameters
-required to validate the challenge.  A client should attempt to fulfill
-one of these challenges, and a server should consider any one of the challenges
+: For pending authorizations, the challenges that the client can
+fulfill in order to prove possession of the identifier.  For valid
+authorizations, the challenge that was validated.  For invalid
+authorizations, the challenges that were attempted and failed.  Each
```

Comment:
I suspect that the text for the "invalid" case could be interpreted multiple ways, but I'm also unsure that it really matters very much.  (That is, does it only include ones that the client responded to but the server failed to validate, or does it include all the challenges that were originally present in the "pending" case?)

Reply:
I would expect it would include all the challenges in the "invalid" state, i.e., ones that the client tried and failed to validate.  But I agree that it doesn't really matter.